components/suggestion/emoticon_provider.tsx

```diff
@@ -142,15 +150,13 @@ export default class EmoticonProvider extends Provider {
                     continue;
                 }
 
-                const matchedArray = recentEmojis.includes(name) ?
-                    recentMatched :
-                    matched;
+                const matchedArray = recentEmojis.includes(name) ? recentMatched : matched;
 
                 matchedArray.push({name, emoji});
             }
         }
 
-        const sortEmojisHelper = (a, b) => {
+        const sortEmojisHelper = (a: any, b: any) => {
```

Comment:
If these parameter types are set to ```SuggestEmoji``` instead of ```any``` as they should be and passed as ```compareEmoji(a.emoji, b.emoji)``` some of the test fails and I can't figure out if I found a bug when type checking was introduced or I made something wrong when converting it.

```compareEmojis``` signature is ```compareEmojis(emojiA: Emoji, emojiB: Emoji, searchedName: string): number``` which I haven't changed

Reply:
I'll try to reproduce this with output for you to look at